lib/views/after_auth_screens/feed/individual_post.dart

```diff
@@ -228,7 +271,13 @@ class CommentTemplate extends StatelessWidget {
   }
 }
 
-/// likedUserCircleAvatar returns a widget of the individual user liked the post.
+/// Circle avatar of users who have liked the post.
+///
+/// **params**:
+/// * `user`: User who liked the post
+///
+/// **returns**:
+/// * `Widget`: returns the circle avatar of the likedBy user who liked the post
```

Comment:
Don't use words like returns, takes etc. It is obvious. Moreover, make the comment multi-line if the single line goes too wide.